src/internal/mersenne/twister.ts

```diff
@@ -322,4 +322,16 @@ export default class MersenneTwister19937 {
     return (a * 67108864.0 + b) * (1.0 / 9007199254740992.0);
   }
   // These real versions are due to Isaku Wada, 2002/01/09
+
+  backupState(): { mt: number[]; mti: number } {
```

Comment:
I would feel safer if we explicitly use `readonly number[]` (or `ReadonlyArray<number>`)

```suggestion
  backupState(): { mt: readonly number[]; mti: number } {
```

Not sure if then something needs to be adjusted, but I assume no 🙂